client/src/pages/CalendarPage/CalendarPage.js

```diff
@@ -98,6 +102,8 @@ const CalendarPage = () => {
   };
 
   useEffect(() => {
+    setToggleError(false);
```

Comment:
This can be moved to the `toggleTaskHandler` function just so it's grouped together with the rest of the toggle related stuff

Reply:
Moved.